docs/azure_jumpstart_arcbox/_index.md

```diff
@@ -227,8 +228,14 @@ After deployment is complete, its time to start exploring ArcBox. Most interacti
 
   ![Screenshot showing Azure Data Studio usage](./azdatastudio.png)
 
+* Open the [ArcBox Azure Monitor workbook](./workbook/_index.md) and explore the visualizations and reports of hybrid cloud resources. A [dedicated README](./workbook/_index.md) is available with more detail on usage of the workbook.
```

Comment:
Need to change these links to the Jumpstart full path

Reply:
by design